test/system/clinic_management_test.rb

```diff
@@ -0,0 +1,133 @@
+require 'application_system_test_case'
+
+# Test workflows around administering lists of clinics a fund works with
+class ClinicManagementTest < ApplicationSystemTestCase
+  before do
+    @clinic = create :clinic, accepts_naf: true,
+                              accepts_medicaid: true
+
+    @user = create :user, role: 'admin'
+    @nonadmin = create :user, role: 'cm'
+    log_in_as @user
+    visit clinics_path
+  end
+
+  describe 'viewing clinic info' do
+    before { visit clinics_path }
+
+    it 'should have existing clinics in the table' do
+      within :css, '#clinics_table' do
+        assert has_text? @clinic.name
+      end
+
+      # TODO evisit this after it's less clunky
```

Comment:
Just noting the TODO. Did you mean to finish this for this PR? If not, maybe throw a followup PR in?

Reply:
Good call! Created an issue for this in https://github.com/DCAFEngineering/dcaf_case_management/issues/1353 and added a bit to the note.